packages/front-end/components/Settings/UpgradeModal/SelfHostedUpgradeForm.tsx

```diff
@@ -70,10 +136,7 @@ export default function SelfHostedUpgradeForm({
               href="https://www.growthbook.io/pricing"
               target="_blank"
               onClick={() => {
-                track("Click Cloud pricing link", {
-                  accountPlan,
-                  source,
-                });
+                track("Self hosted: Click Cloud pricing link", customerContext);
```

Comment:
Same here re tracking. For the event name, I would do `Click Cloud Pricing Link`